helm/upp-aggregate-healthcheck/templates/admin-ingress.yaml

```diff
@@ -0,0 +1,24 @@
+{{ if eq .Values.ingress.enabled "true" }}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{.Values.service.name}}-admin-ingress
```

Comment:
The admin-ingress' are the `/__service-name` style endpoints, so I'd rename this

Reply:
Renamed as operation-ingrees